pennylane/devices/qutrit_mixed/measure.py

```diff
@@ -36,26 +36,33 @@
 
 
 def calculate_expval(
-    measurementprocess: ExpectationMP, state: TensorLike, is_state_batched: bool = False
+    measurementprocess: ExpectationMP,
+    state: TensorLike,
+    is_state_batched: bool = False,
+    measurement_errors=None,
 ) -> TensorLike:
     """Measure the expectation value of an observable.
 
     Args:
         measurementprocess (ExpectationMP): measurement process to apply to the state.
         state (TensorLike): the state to measure.
         is_state_batched (bool): whether the state is batched or not.
+        measurement_errors (List[Callable]): List of operators to apply to each wire being measured
+        to simulate measurement error.
 
     Returns:
         TensorLike: expectation value of observable wrt the state.
     """
-    probs = calculate_probability(measurementprocess, state, is_state_batched)
+    probs = calculate_probability(measurementprocess, state, is_state_batched, measurement_errors)
     eigvals = math.asarray(measurementprocess.eigvals(), dtype="float64")
     # In case of broadcasting, `probs` has two axes and these are a matrix-vector products
     return math.dot(probs, eigvals)
 
 
 def calculate_reduced_density_matrix(
-    measurementprocess: StateMeasurement, state: TensorLike, is_state_batched: bool = False
+    measurementprocess: StateMeasurement,
```

Comment:
question (non-blocking): why did the formatting change here?

Reply:
I think it may be the black version, I'm not entirely sure.